test/main.js

```diff
@@ -23,14 +21,10 @@ function init(callback) {
 	require('./../app');
 
 	// Run the battle engine in the main process to keep our sanity
-	let BattleEngine = global.BattleEngine = require('./../battle-engine');
 	for (let listener of process.listeners('message')) {
 		process.removeListener('message', listener);
 	}
```

Comment:
@Slayer95 I think that might be out of scope for this PR but definitely a good idea to remove it in that case.